skypy/halos/tests/test_sampler.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
+from scipy import integrate
+from scipy.stats import kstest
+import pytest
+
+
+@pytest.mark.flaky
+def test_colossus_mass_sampler():
+    from astropy.cosmology import WMAP9
```

Comment:
```suggestion
    from astropy.cosmology import WMAP9
    import colossus as colossus
    from colossus.lss import mass_function
```
import colossus inside the test so that no ImportErrors are raised if it is not installed